### PR TITLE
Feature/heading setting

### PR DESCRIPTION
**Pull request by MikaelNordberg**
New setting that controls the headings (H1/H2) in PxWeb

### CODE REVIEW

File: PXWeb/Code/Settings/SiteSettings.cs [hunk 1]
```diff
@@ -31,11 +31,13 @@ public SiteSettings(XmlNode siteNode)
 
             //Load the Site settings 
             xpath = "./applicationName";
-            ApplicationName = SettingsHelper.GetSettingValue(xpath, siteNode, "PX-Web");
+            ApplicationName = SettingsHelper.GetSettingValue(xpath, siteNode, "PxWeb");
```
Comment (JohannesFinsveen) [inline PXWeb/Code/Settings/SiteSettings.cs:35]: I can't spot the same change for writing to the file ...

File: PXWeb/Code/CustomControls/HeadingLabel.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            H1,
+            H2,
+            H3,
+            H4,
+            H5,
+            H6
+        }
+
+        public HeadingLevel Level { get; set; } = HeadingLevel.H1;
+
```
Comment (JohannesFinsveen) [inline PXWeb/Code/CustomControls/HeadingLabel.cs:21]: Nice!  = default;

File: PXWeb/Resources/Styles/main-pxweb.css [hunk 3]
```diff
@@ -106,14 +106,36 @@ $scrollbar-color: #b0b0b0;
 /* **********************************  _PxWeb.scss *********************************** */
 h1 {
```
Comment (JohannesFinsveen) [inline PXWeb/Resources/Styles/main-pxweb.css:107]: The pure element selectors could perhaps go in the _global-styles-on-elements.scss , but perhaps not :-)

**Review by likp — APPROVED** [review]
Looks ok

**Review by JohannesFinsveen — APPROVED** [review]

**Review by PerIngeVaaje — APPROVED** [review]
OK.

**Pull request merged**